source/jormungandr/jormungandr/planner.py

```diff
@@ -33,7 +33,10 @@ def __init__(self,
                  walking_transfer_penalty=120,
                  direct_path_duration=None,
                  night_bus_filter_max_factor=None,
-                 night_bus_filter_base_factor=None):
+                 night_bus_filter_base_factor=None,
+                 min_nb_journeys=None,
```

Comment:
😳 thanks for fixing `min_nb_journeys` with experimental

Reply:
:rabbit: